modules/claims_api/app/controllers/claims_api/v2/blueprints/transformers/lower_camel_transformer.rb

```diff
@@ -6,7 +6,7 @@ module Blueprints
       module Transformers
         class LowerCamelTransformer < Blueprinter::Transformer
           def transform(hash, _object, _options)
-            hash.transform_keys! { |key| key.to_s.camelize(:lower).to_sym }
+            hash.transform_keys! { |key| key.to_s == 'POA' ? key : key.to_s.camelize(:lower).to_sym }
```

Comment:
I don't love this, but otherwise this was transforming `POA` to `pOA`. Thoughts on making the field lowercase `poa`, or is it semantically better to have is uppercase as an acronym?

Reply:
I would think allowing it to be all lowercase would be fine.